coralproject/talk
# CORL-2083: text for next available comment download

## What does this PR do?



## What changes to the GraphQL/Database Schema does this PR introduce?

None.

## How do I test this PR?

Enable downloading of comments. Then as a commenter, go to "My Profile" and request a download of your comments. Note that the message references a date 2 weeks in the future, where it once referenced the time at which you submitted the request.
